specification/devcenter/resource-manager/Microsoft.DevCenter/preview/2022-08-01-preview/vdi.json

```diff
@@ -0,0 +1,1498 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-08-01-preview",
+    "title": "DevCenter",
```

Comment:
```suggestion
    "title": "DevCenterClient",
```

Reply:
adopt this suggestion if you want all language SDK to have a client class name `DevCenterClient`